tests/tizen_capi/unittest_tizen_capi.cc

```diff
@@ -1836,14 +1838,16 @@ TEST (nnstreamer_capi_singleshot, invoke_02)
   int status;
 
   const gchar *root_path = g_getenv ("NNSTREAMER_BUILD_ROOT_PATH");
-  gchar *test_model;
+  gchar *tmp, test_model[MAX_PATH_LEN];
```

Comment:
I knew this means that
```c
gchar (*tmp), (test_model[MAX_PATH_LEN]);
```
However, for the readability, please consider
```c
gchar test_model[MAX_PATH_LEN];
gchar *tmp;
```

> It’s also important to comment data, whether they are basic types or derived types. To this end, use just one data declaration per line (no commas for multiple data declarations). This leaves you room for a small comment on each item, explaining its use. 
from https://www.kernel.org/doc/html/v4.10/process/coding-style.html

I think this aims not just room for comment but also the readability of code.